_posts/2019-01-06-dddem-post.md

```diff
@@ -27,7 +27,7 @@ The East Midlands has a lot to boast about in term of the tech companies, talent
 
 <br/>
 
-# What has DDD got to do with it?
```

Comment:
DDD is the umbrella organisation that we are a subsidiary of. I'm sorry but this change doesn't quite make sense with the blog post. (Also the title is a mickey take of a pop song)